osdk/src/util.rs

```diff
@@ -16,11 +16,13 @@ use quote::ToTokens;
 /// and use the published version in the generated Cargo.toml.
 pub const ASTER_GIT_LINK: &str = "https://github.com/asterinas/asterinas";
 /// Make sure it syncs with the builder dependency in Cargo.toml.
-pub const ASTER_GIT_REV: &str = "ccc4e6e";
+/// FIXME: We cannot use `include_str!("../../VERSION")` here
+/// because cargo publish does not allow use files outside of the crate directory.
+pub const ASTER_GIT_TAG: &str = "v0.5.0";
```

Comment:
Since OSDK have it's version synced with Aster, you can use `env!("CARGO_PKG_VERSION")` here.